src/services/branch-manager.js

```diff
@@ -6,8 +16,53 @@ async function deleteBranch() {
   // FIXME: Implement deleteBranch function
 }
 
-async function createBranch() {
-  // FIXME: Implement createBranch function
+async function createBranch(branchName) {
+  const authToken = authenticator.getAuthToken();
+
+  await agent
+    .post(`${serverHost()}/api/branches`)
+    .set('Authorization', `Bearer ${authToken}`)
+    .set('forest-secret-key', `${process.env.FOREST_ENV_SECRET}`)
+    .send({ branchName });
+}
+
+function handleError(error) {
+  try {
+    const { status } = error;
+
+    if (status === 404) {
+      // NOTICE: When no env/project can be found through envSecret
+      return ERROR_MESSAGE_ENV_SECRET_ISSUE;
+    }
+
+    const message = JSON.parse(error.response.text).errors[0].detail;
```

Comment:
[Suggestion] Since we'll have to work on other commands, can't we create an `error` deserializer ?